core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -652,6 +656,10 @@ public synchronized void close() {
         // If the tailing thread has crashed before the close,
         // an exception will be thrown, containing what has originally caused the crash
         mTailerThread.awaitTermination(false);
+        if (mState.get() == State.STANDBY) {
+          LOG.info("Closing journal {}, state {}, next sequence number {}",
+              this, this.mState, mTailerThread.getNextSequenceNumber());
```

Comment:
sanity check on the null check, could `mTailerThread` be dead (and null) already?

Reply:
if (mTailerThread != null) {